src/analysis/imports_sortedness.d

```diff
@@ -66,7 +66,7 @@ class ImportSortednessCheck : BaseAnalyzer
 
 private:
 
-	int level = 0;
```

Comment:
TBH, despite the recent discussion, I find code without explicit initializers (when equal to .init) more clear to read, so LGTM.